### PR TITLE
Add Icon, Progressbar, Logo and a lot of things

### PR DESCRIPTION
**Pull request by HorstFrank**
Unfortunately it was impossible to pack the many changes into single branches. We have to live with a single pull request for this time. I am very, very sorry. honestly.

### CODE REVIEW

File: components/Button/Button2.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  label: "Beschriftung 3",
+};
+
+export const Secondaryx = Template.bind({});
+Secondaryx.args = {
+  label: "BEschriftung 2",
+};
+
+// https://storybook.js.org/docs/react/api/csf
+// https://storybook.js.org/docs/react/writing-docs/doc-blocks#argstable
```
Comment (fre-ben) [inline components/Button/Button2.stories.tsx:25]: Remove comments

File: components/Button/Button2.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export const Primaryx = Template.bind({});
+Primaryx.args = {
+  primary: true,
+  label: "Beschriftung 3",
+};
+
+export const Secondaryx = Template.bind({});
+Secondaryx.args = {
+  label: "BEschriftung 2",
```
Comment (fre-ben) [inline components/Button/Button2.stories.tsx]:
```suggestion
  label: "Beschriftung 2",
```

File: components/Button/Button2.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/*
+
+THIS IS A SAMPLE COMPONENT
+
+*/
```
Comment (fre-ben) [inline components/Button/Button2.tsx:5]: Could remove this comment also :)

File: components/Progressbar/Progressbar.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+import { Story, Meta } from "@storybook/react/types-6-0";
+import Progressbar from "./Progressbar";
+
+export default {
+  title: "Common/SomeStuff",
+  component: Progressbar,
+} as Meta;
+
+// export const progressbar = () => <Progressbar />;
```
Comment (fre-ben) [inline components/Progressbar/Progressbar.stories.tsx:12]: Remove comment

File: components/Progressbar/Progressbar.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  colorBackground: "gray",
+  progress: 0.5,
+  width: "100%",
+  height: 2,
+};
+
+// export const Secondaryx = Template.bind({});
+// Secondaryx.args = {
+//   label: "BEschriftung 2",
+// };
```
Comment (fre-ben) [inline components/Progressbar/Progressbar.stories.tsx:31]: Remove comment

File: components/icons/iconSettings.tsx [hunk 6]
```diff
@@ -1,12 +1,27 @@
-import "./iconSettings.module.css";
+import styles from "./IconSettings.module.css";
+import { IconType } from "../../types/IconType";
+// import { SVGProps } from "react";
```
Comment (fre-ben) [inline components/icons/iconSettings.tsx:3]: Remove comment

File: components/icons/iconSettings.tsx [hunk 7]
```diff
@@ -1,12 +1,27 @@
...
+    stroke: props.color || props.stroke || "black",
+    strokeLinecap: props.strokeLinecap || "round",
+    strokeLinejoin: props.strokeLinejoin || "round",
+    strokeWidth: props.strokeWidth || 1,
+  };
 
-export default function iconSettings() {
-  return <>fdgsdfgsdfg</>;
+  return (
+    // <svg width="100%" height="100%" >
```
Comment (fre-ben) [inline components/icons/iconSettings.tsx:15]: Remove comment

File: components/logo/Logo.tsx [hunk 8]
```diff
@@ -0,0 +1,113 @@
+// import { SVGProps } from "react";
+
+/*
+memo:
+SVGAttributes<T>.style?: CSSProperties
+*/
```
Comment (fre-ben) [inline components/logo/Logo.tsx:6]: Remove comments

**Review by fre-ben — APPROVED** [review]
LGTM 🤖 
Lots of commented out code you could remove before creating a PR.
Also do you need the font description files?

//Edit: PR title is not really descriptive 🤕

File: components/Button3/Button2.stories.tsx [hunk 9]
```diff
@@ -20,3 +20,6 @@ export const Secondaryx = Template.bind({});
 Secondaryx.args = {
   label: "BEschriftung 2",
```
Comment (m-wiemers) [inline components/Button3/Button2.stories.tsx]:
```suggestion
  label: "Beschriftung 2",
```

**Review by m-wiemers — APPROVED** [review]
There are flowers that are beautiful 🌹. There are women who are beautiful 💃. And there is code from Boris. It's so really beautiful 👨‍💻

**Pull request merged**